scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java

```diff
@@ -307,40 +320,53 @@ public JobId submit(URL job)
     }
 
     @Override
-    public JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName)
+    public JobId submit(URL job, Map<String, String> variables)
+            throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
+        return submit(null, job, variables);
+    }
+
+    @Override
+    public JobId submit(Map<String, String> genericInfos, URL job, Map<String, String> variables)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
         renewSession();
-        return client.submitFromCatalog(catalogRestURL, bucketName, workflowName);
+        return client.submit(job, variables, genericInfos);
     }
 
     @Override
     public JobId submit(URL job, Map<String, String> variables, Map<String, String> headerParams)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
+        return submit(job, variables, null, headerParams);
+    }
+
+    @Override
+    public JobId submit(URL job, Map<String, String> variables, Map<String, String> genericInfos,
+            Map<String, String> headerParams)
+            throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
         if (client == null)
             throw new NotConnectedException("Client not connected, call connect() before using the scheduler client");
-        return this.client.submit(job, variables, headerParams);
+        return this.client.submit(job, variables, genericInfos, headerParams);
     }
 
     @Override
-    public JobId submit(File job, Map<String, String> variables)
+    public JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)